src/router/routes.js

```diff
@@ -142,30 +124,7 @@ module.exports = [
 		path: '/covid19response',
 		redirect: '/monthlygood'
 	},
-	{
-		path: '/design',
-		component: () => import('@/pages/ContentfulPage'),
-		meta: {
-			contentfulPage: () => 'design',
-		},
-	},
 	{ path: '/donate/support-kiva', component: () => import('@/pages/Donate/DonateFromMacro') },
-	{
-		path: '/donate/supportus',
```

Comment:
The 2 `support` pages, here and below, don't use `@/pages/ContentfulPage` so they cannot be removed at this time.